### ISSUE TITLE
Disruptions view

### ISSUE THREAD

**Issue by samijuhani** [HSLdevcom/transitlog-ui#75]
Show all disruption in a view that is opened from https://github.com/HSLdevcom/transitlog-ui/issues/43 

First there are disruptions considering given journey. Below that all other general disruption that time or day.

_Poikkeuksen   tyyppi
--
Syy   ylätaso
Syy   alataso
Otsikko
Kuvaus
Viimeksi   muokattu ja muokkaaja
Kaikki   poikkeaman lähdöt: (linja, suunta, pvm, lähtöaika, ilmoitettu, reitin kuvaus,   liikennöitsijä, kilpailukohde, sanktiokerroin)
Tilaajan   ja liikennöitsijän tilat
Aiemmat   tilat
Kommentit_

**Comment by samijuhani** [comment]
Waiting for GTFS-RT to be in production. Implementation asap after that.

**Comment by samijuhani** [comment]
Also passenger notices considering same line should be shown.

**Comment by samijuhani** [comment]
Idea: Could "Homepage" always show disruptions and service alerts of chosen time by default when nothing is selected yet and user could click disruption and linker journeys straight from there. Could be easy way to see known problems before even searching anything.

**Comment by danieldunderfelt** [comment]
@samijuhani yes it could! That's a good idea. It would also show the weather as a larger element since it doesn't require anything else except the time.
